Patches/Nearmare Race/ThingDefs_Misc/Weapons.xml

```diff
@@ -325,10 +325,88 @@
 					</tools>
 				</value>
 			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Gun_AssaultRifle_NM"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>stock</label>
+							<capacities>
+								<li>Blunt</li>
+							</capacities>
+							<power>8</power>
+							<cooldownTime>1.55</cooldownTime>
+							<chanceFactor>1.5</chanceFactor>
+							<armorPenetrationBlunt>2.755</armorPenetrationBlunt>
+							<linkedBodyPartsGroup>Stock</linkedBodyPartsGroup>
+						</li>
+						
+						<li Class="CombatExtended.ToolCE">
+							<label>barrel</label>
+							<capacities>
+								<li>Blunt</li>
+							</capacities>
+							<power>5</power>
+							<cooldownTime>2.2</cooldownTime>
+							<armorPenetrationBlunt>1.630</armorPenetrationBlunt>
+							<linkedBodyPartsGroup>Barrel</linkedBodyPartsGroup>
+						</li>
+						
+						<li Class="CombatExtended.ToolCE">
+							<label>muzzle</label>
+							<capacities>
+								<li>Poke</li>
+							</capacities>
+							<power>8</power>
+							<cooldownTime>1.55</cooldownTime>
+							<armorPenetrationBlunt>2.755</armorPenetrationBlunt>
+							<linkedBodyPartsGroup>Muzzle</linkedBodyPartsGroup>
+						</li>
+					</tools>
+				</value>
+			</li>
 			
+			<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+			<defName>Gun_AssaultRifle_NM</defName>
+				<statBases>
+					<SightsEfficiency>1</SightsEfficiency>
+					<Bulk>6.65</Bulk>
+					<ShotSpread>1</ShotSpread>
+					<SwayFactor>1.3</SwayFactor>
+					<Mass>3.5</Mass>
+					<RangedWeapon_Cooldown>0.36</RangedWeapon_Cooldown>
+				</statBases>
+				
+				<Properties>
+					<recoilAmount>3</recoilAmount>
+					<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Bullet_127x55mmAR_FMJ</defaultProjectile>
+					<warmupTime>5</warmupTime>
+					<range>52</range>
+					<burstShotCount>10</burstShotCount>
+					<ticksBetweenBurstShots>2</ticksBetweenBurstShots>
+					<soundCast>Shot_AssaultRifle</soundCast>
+					<soundCastTail>GunTail_Medium</soundCastTail>
+					<muzzleFlashScale>11</muzzleFlashScale>
+				</Properties>
+				
+				<AmmoUser>
+					<magazineSize>45</magazineSize>
+					<reloadTime>10</reloadTime>

```

Comment:
This is a very long reload for a magazine fed assault rifle. For comparison, the vanilla Assault rifle reloads in 4.6. If you wanted this to take a bit longer to reload, I'd set the time to around 5.2.

Reply:
Ok. I will change it.